Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -0,0 +1,150 @@
+import UIKit
+import BraintreeCore
+import BraintreePayPal
+import BraintreeVenmo
+import BraintreeShopperInsights
+
+class ShopperInsightsViewController: PaymentButtonBaseViewController {
```

Comment:
Not sure if we want to 🏈  on this based on the PR description but subclassing `PaymentButtonBaseViewController` should allow us to get a lot of this for "free". Specially the `createButton` and ability to remove the override of the auth init.

Reply:
I know this is in the Demo app, so not as merchant impacting, but I personally think it's acceptable to take a little time and do this work now. If folks push back we can lean on the company's leadership principles.

Reply:
This is a great callout! Addressed in 92676b675c06beb21269bc7a19b94d9b8565525e